.github/workflows/pr.yaml

```
@@ -92,6 +92,34 @@ jobs:
         run: |
           composer coding-standards-check
 
+  coding-standards-twig-cs-fixer:
+    name: Twig - Check Coding Standards
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup Twig, with composer and extensions
+        uses: vincentlanglet/twig-cs-fixer
+        with:
+          coverage: none
+          tools: composer:v2
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ runner.os }}-composer-
+      - name: Install Dependencies
+        run: |
+          composer install --no-interaction --no-progress --no-scripts
+          # @see https://github.com/Dealerdirect/phpcodesniffer-composer-installer#calling-the-plugin-directly
+          composer run-script install-codestandards
+      - name: TwigCS
+        run: |
+          composer coding-standards-check
```

You should only check Twig files here:

```suggestion
          composer coding-standards-check/twig-cs-fixer
```

Similarly, line 93 should be updated to

```yaml
          composer coding-standards-check/phpcs
```

to only check PHP.